app/views/static/cloud_subnet/cloud-subnet-form.html.haml

```diff
@@ -0,0 +1,130 @@
+                'ng-model'                    => 'vm.cloudSubnetModel.network_id',
+                'ng-options'                  => 'network.ems_ref as network.name for network in vm.available_networks',
+                'pf-select'                   => true,
+                'required'                    => '',
+                'selectpicker-for-select-tag' => ''}
+          %option{'value' => ''}
+            = "<#{_('Choose')}>"
+        %span.help-block{'ng-show' => 'angularForm.network_id.$error.required'}
+          = _('Required')
+      .col-md-8{'ng-if' => '!vm.newRecord'}
+        %input.form-control{'type'      => 'text',
+                            'name'      => 'network_name',
+                            'maxlength' => 128,
+                            'ng-model'  => 'vm.cloudSubnetModel.cloud_network.name',
+                            'disabled'  => true}
+
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet Name')
+      .col-md-8
+        %input.form-control{'type'      => 'text',
+                            'name'      => 'name',
+                            'ng-model'  => 'vm.cloudSubnetModel.name',
+                            'maxlength' => 128}
+    .form-group
+      %label.col-md-2.control-label
+        = _('Gateway')
+      .col-md-8
+        %input.form-control{'type'      => 'text',
+                            'name'      => 'gateway_ip',
+                            'ng-model'  => 'vm.cloudSubnetModel.gateway',
+                            'maxlength' => 20}
+    .form-group
+      %label.control-label.col-md-2{'for' => 'dhcp'}
+        = _('DHCP')
+      .col-md-8
+        %input{'type'        => 'checkbox',
+               'id'          => 'dhcp',
+               'name'        => 'dhcp',
+               'bs-switch'   => '',
+               'data'        => {:on_text => _('Enabled'), :off_text => _('Disabled'), :size => 'mini'},
+               'ng-model'    => 'vm.cloudSubnetModel.dhcp_enabled'}
+    .form-group{'ng-class' => '{"has-error": angularForm.network_protocol.$invalid}'}
+      %label.col-md-2.control-label
+        = _('IP Version')
+      .col-md-8
+        %select{'name'                        => 'network_protocol',
+                'ng-model'                    => 'vm.cloudSubnetModel.network_protocol',
+                'ng-disabled'                 => '!vm.newRecord',
+                'ng-options'                  => 'protocol as protocol for protocol in vm.networkProtocols',
+                'pf-select'                   => true,
+                'required'                    => '',
+                'selectpicker-for-select-tag' => ''}
+        %span.help-block{'ng-show' => 'angularForm.network_protocol.$error.required'}
+          = _('Required')
+
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet CIDR')
+      .col-md-8
+        %input.form-control{'type'        => 'text',
```

Comment:
Bad rebase - https://github.com/ManageIQ/manageiq-ui-classic/pull/3769 made this field required

Reply:
Fixed, thanks